docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -118,8 +118,8 @@ lifecycle:
 | 2.69.0         | January 2025     |                  | Minor        | STS          | 2.70.0                  |
 | 2.70.0         | February 2025    |                  | Minor        | STS          | 2.71.0                  |
 | 2.71.0         | March 2025       |                  | Minor        | STS          | 2.72.0                  |
-| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
-| 2.72.0         | May 2025         | Yes              | Major        | LTS          | 2.73.0                  |
+| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.73.0**_            |
```

Comment:
```suggestion
| 2.72.0         | April 2025       |                  | Minor        | STS          | 2.73.0            |
```
why would 2.73 be in bold since it is a regular release?